### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Abenezer-Tilahun**
### Description
In this PR I have updated views for user and posts using the following routes:
- users
- users/:id
- users/:id/posts
- users/:id/posts/:id

### General requirements
- There are no linter errors.
- Followed the Github flow.
- The documentation is professional.

### Ruby requirements
- Followed [best practices of Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md)

### Project requirements

- [x] Implement the design from the sneak peek wireframes.
- Styling is nice to have but focus on making sure everything functions first.
- [x] Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- [x] Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data)

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+  <div class="card text-dark bg-light m-2 d-flex flex-row" style="max-width: 600px;">
+    <div class="col-md-4  d-flex align-items-center">
+      <h5 class="card-title"> <%= link_to user.name, users_url(user), class: 'display-5 text-primary' %></h5> 
+    </div>
+
+    <div class = " col-md-7 d-flex align-items-end justify-content-end" >
+      <p class="text-right">Number of posts: 
+        <%= user.posts_counter ?  user.posts_counter : 0   %>
+        </p>
+      </p>
```
Comment (sja-thedude) [inline app/views/users/index.html.erb]: Currently, your app is not according to the design provided -
- [x] Kindly make sure to follow the wireframe as [given](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md).
- [x] Make sure that when the `User name` is clicked all posts are displayed. 
- [x] Make sure that When `see all posts` button is clicked all posts with comments are shown.
- [x] Make sure that when one post is clicked all the comments of that specific post are shown.

____________________________________________

- Kindly look below when I clicked user name I am not able to view the required page.

https://user-images.githubusercontent.com/70269507/173761628-20352e46-da4c-41d5-a711-3c5ff6b94211.mov

- As you can see below I have tried adding users, posts and comments but still it's not displayed.

Screenshot 2022-06-15 at 9 28 12 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Abenezer-Tilahun, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good PR title and description
✔️ Professional readme
✔️ Correct git-flow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (Abenezer-Tilahun) [reply app/views/users/index.html.erb]: Hello @sja-thedude Thanks for suggestion I fix the issues
sample-1.
sample-2
sample-3

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @Abenezer-Tilahun 👋🏻, 

Good job making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**